testsuite/python3.py

```diff
@@ -13,6 +13,16 @@ def foo(x: int) -> int:
 
 class Class:
     cls_var: ClassVar[str]
-
+    for_var: ClassVar[str]
+    while_var: ClassVar[str]
+    def_var: ClassVar[str]
+    if_var: ClassVar[str]
+    elif_var: ClassVar[str]
+    else_var: ClassVar[str]
+    try_var: ClassVar[str]
+    except_var: ClassVar[str]
+    finally_var: ClassVar[str]
+    with_var: ClassVar[str]
```

Comment:
Might as well be *paranoid* and add camelCased versions of each of these while we're here.